net/jitsi-meet-electron/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        jitsi jitsi-meet-electron 2023.9.0 v
+github.tarball_from archive
+revision            0
+
+categories          net
+installs_libs       no
+license             Apache-2
+maintainers         {takikawa @takikawa} openmaintainer
+description         Jitsi Meet Electron
+long_description    Jistsi Meet Electron is a client for the open-source Jitsi Meet teleconferencing system
+
+checksums           rmd160  5d7cba90c04e4fa5f4fa4b2a675c4a420866b49f \
+                    sha256  6fcc829b89e91bba66d8185bc88d2fbe4bb734a885b97fa3c20965155ff04983 \
+                    size    827991
+
+depends_build       port:yarn
+
+build.env-append    CSC_IDENTITY_AUTO_DISCOVERY=false
+
+use_configure       no
+use_xcode           yes
```

Comment:
this needs a full XCode installation or is an CLT installation sufficient?

Reply:
I think it doesn't need a full xcode install, so I've removed this line in the new version. It still builds so it seems ok.